internal/server/data/organization_test.go

```diff
@@ -0,0 +1,57 @@
+package data
+
+import (
+	"reflect"
+	"testing"
+
+	gocmp "github.com/google/go-cmp/cmp"
+	"gotest.tools/v3/assert"
+
+	"github.com/infrahq/infra/internal/server/models"
+)
+
+var cmpOrgShallow = gocmp.Comparer(func(x, y models.Organization) bool {
```

Comment:
Shallow comparisons are great for testing things like `List` where we don't care too much about every field, but for `Create` I think we need to make sure every field is checked. 

I think you can use `cmpModel` (in `settings_test.go`) to handle the fields we can't predict.